src/pages/resultsView/plots/PlotsTab.spec.tsx

```diff
@@ -4,6 +4,7 @@ describe("PlotsTab", ()=>{
     describe("controls", ()=>{
         describe("gene lock", ()=>{
             it("genes can be freely selected when gene lock is off", ()=>{
+                assert.equal(1,1);
```

Comment:
hmm!

Reply:
Wooops, leftover from other code changes. Sorry for this. File is reverted back to its original state.